packages/rum-core/src/domain/configuration.ts

```diff
@@ -156,6 +261,9 @@ export function serializeRumConfiguration(configuration: RumInitConfiguration):
       action_name_attribute: configuration.actionNameAttribute,
       use_allowed_tracing_origins:
         Array.isArray(configuration.allowedTracingOrigins) && configuration.allowedTracingOrigins.length > 0,
+      use_configure_tracing_urls:
+        Array.isArray(configuration.configureTracingUrls) && configuration.configureTracingUrls.length > 0,
+      selected_tracing_headers: getSelectedTracingHeaders(configuration),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You should update rum-events-format to add those new fields for telemetry configuration